python-iml-manager.spec

```diff
@@ -294,6 +294,11 @@ fi
 mkdir -p %{_localstatedir}/lib/rpm-state/%{name}/
 cp -r /var/lib/chroma/* %{_localstatedir}/lib/rpm-state/%{name}/
 
+%posttrans -n python2-%{pypi_name}-libs
+if [ -d %{python_sitelib}/iml_manager-5.0.*-py2.7.egg-info.egg-info ]; then
+    rm -rf %{python_sitelib}/iml_manager-5.0.*-py2.7.egg-info.egg-info
+fi
```

Comment:
Should this just be ```rmdir %{python_sitelib}/iml_manager-5.0.*.egg-info.egg-info```?
That way it cleans up empty directories but doesn't remove anything that is installed vis rpm?

Reply:
Sure

Reply:
Sure